theme/js/components/dataset/resource/resource.vue

```diff
@@ -146,7 +146,7 @@
                   <div class="fr-grid-row fr-grid-row--middle fr-mb-1w">
                     <p class="fr-text--xs fr-m-0">{{$t('This file is following a schema: ')}}</p>
                     <p class="fr-tag fr-tag--sm fr-icon-checkbox-circle-line fr-tag--icon-left">
-                      {{resource.schema.name}}
+                      {{resource.schema.name ? resource.schema.name : resource.schema.url}}
```

Comment:
There is also a `See schema documentation` button later on that should be displayed only when resource.schema.name :)